tasks/agent.py

```diff
@@ -189,6 +190,8 @@ def refresh_assets(ctx, build_tags, development=True, puppy=False):
         copy_tree("./cmd/agent/dist/conf.d/{}.d/".format(check), check_dir)
     if "apm" in build_tags:
         shutil.copy("./cmd/agent/dist/conf.d/apm.yaml.default", os.path.join(dist_folder, "conf.d/apm.yaml.default"))
+    if "process" in build_tags:
```

Comment:
This won't be used (in fact, I don't think we should do it for APM, but that's another story)

Reply:
Completely disregard my review; it's not correct for OSX